tests/core/editor/manual/customconfigrace.md

```diff
@@ -0,0 +1,14 @@
+@bender-tags: bug, 4.14.2, 3361
+@bender-ui: collapsed
+@bender-ckeditor-plugins: toolbar, wysiwygarea, basicstyles
+
+Verify editors width.
+
+## Expected
+
+* Editor 1 and Editor 3 have the same width (`200px`).
+* Editor 2 have different width (`400px`).
```

Comment:
If you agree with me on the change of `configfile` used in the test, this should be changed to `300px`.